recipes/tvips-tools/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "TVIPS tools" %}
+{% set tarname = name|lower|replace(' ', '-') %}
+{% set version = "0.0.0" %}
+
+package:
+  name: {{ tarname }}
+  version: {{ version }}
+
+source:
+  url: https://cryoem.ucla.edu/downloads/{{ tarname }}-{{ version }}.tar.gz
+  sha256: 87b5416470e4fa93b08c97ae8c73f71552945058e7f3a963e64248530646403d
+  patches:
+    - 01-setenv.patch
+    - 02-version.patch
+    - 03-warnings.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - coreutils
+    - make
+    - libxslt >=0.13.0
+  host:
+    - libtiff 4.*
+  run:
+    - {{ pin_compatible('libtiff', max_pin='x.x') }}
+
+test:
+  commands:
+    - dm2smv -V
+    - tiff2smv -V
+    - tvips2smv -V
+
+about:
+  home: https://cryoem.ucla.edu/microed
+  license: ISC
+  license_file: LICENSE
+  summary: "Convert diffraction images from TVIPS format to SMV"
+  description: |
+    The TVIPS tools convert sequences of related electron diffraction
+    images, or MicroED sweeps, recorded by Tietz Video and Image
+    Processing Systems (TVIPS) cameras to the Super Marty View (SMV)
+    format.
+  dev_url: https://github.com/gonenlab/microed-tools
```

Comment:
This URL is not valid. Maybe the repo is private or was moved?

Reply:
Removed optional dev_url.